_layouts/post.html

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+---
+
+<section class="articles pdt-50 pdb-50">
+  <div class="column is-10 is-offset-1">
+    <div class="card article">
+      <div class="card-content">
+        <div class="media">
+          <div class="media-content has-text-centered">
+            <p class="title article-title">{{ page.title}}</p>
+            <p class="subtitle is-6 article-subtitle">
+              <a href="https://twitter.com/{{page.author-twitter}}">{{ page.author }}</a> on {{ page.date | date: "%-d %B, %Y"}}
+            </p>
+          </div>
+        </div>
+        <div class="content article-body">
+          <p>{{ content }}</p>
+        </div>
+      </div>
+      {% include author-category-box.html %}
```

Comment:
```{% include author-category-box.html %}``` won't work here !

Reply:
They have to be in the `_includes` folder
See 
1. https://jekyllrb.com/docs/includes
2. https://github.com/RubyNepal/rubynepal.github.io/tree/source/source/_includes

Reply:
That was my mistake in last hour for reusing the box. I will make correct it. Actually in file ```author-category-box.html``` contains post variable, but in _layouts/post.html  there is page variable.